### ISSUE TITLE
Installation stuck at "waiting on database connection"

### ISSUE THREAD

**Issue by HelLViS69** [passbolt/charts-passbolt#89]
Hello,
as title says the installation remains stuck at "waiting on database connection".

After entering the pod "passbolt-depl-srv" and open a shell in the init container "passbolt-depl-srv-init", I tried the connection to MariaDB and this is the result:

```bash
root@passbolt-depl-srv-5cf966745f-vblgt:/# mariadb -h ${DATASOURCES_DEFAULT_HOST} -u ${DATASOURCES_DEFAULT_USERNAME} -p${DATASOURCES_DEFAULT_PASSWORD}
ERROR 2026 (HY000): TLS/SSL error: SSL is required, but the server does not support it
```

I also tried to pass PASSBOLT_SSL_FORCE to false but didn't help

**Comment by dlen** [comment]
Looks like your database server requires SSL to connect, right?

If that is the case you'll need to configure passbolt to use the SSL certificates to communicate with your database. Check the following env vars: https://github.com/passbolt/passbolt_api/blob/727e938c35dbbd43f798f36bbfba25898d8ae7b5/config/app.default.php#L343C13-L345C63

You'll have to mount the certificates in your passbolt container using the extraVolumes and extraVolumeMounts. Then passbolt should be able to connect.

If you want to use the mariadb cli you'll have to pass certain parameters to use SSL https://mariadb.com/kb/en/securing-connections-for-client-and-server/

**Comment by dlen** [comment]
Please share your values.yaml

**Comment by HelLViS69** [comment]
I'm using K3s, so the values.yaml is wrapped in the following CRD:

```yaml
apiVersion: helm.cattle.io/v1
kind: HelmChartConfig
metadata:
  namespace: kube-system
  name: passbolt
spec:
  valuesContent: |-
    redis:
      auth:
        password: "xxx"
    mariadb:
      auth:
        username: "passbolt"
        password: "yyy"
        replicationPassword: "zzz"
    passboltEnv:
      plain:
        APP_FULL_BASE_URL: "https://passbolt.local"
        PASSBOLT_KEY_EMAIL: "passbolt@yourdomain.com"
        PASSBOLT_SSL_FORCE: false
        EMAIL_DEFAULT_FROM: "no-reply@passbolt.local"
      secret:
        CACHE_CAKE_DEFAULT_PASSWORD: "xxx"
        DATASOURCES_DEFAULT_USERNAME: "passbolt"
        DATASOURCES_DEFAULT_PASSWORD: "yyy"
```

I just saw that there's an additional configuration I didn't try yet:
```yaml
    app:
      tls:
        autogenerate: false
        existingSecret: ""
```

This could be af any help?

**Comment by schulmar** [comment]
I did also see this. However, I am not sure if the CLI command is indicative of what is happening on the actual service's connection. Anyways, I got it to work with `mariadb --skip-ssl ...`.

**Comment by abnerrizzi** [comment]
on templates/deployment.yaml, on line 64, add --skip-ssl
change from:
while ! ${client} --defaults-file=/tmp/defaultsfile.cnf -e "SELECT 1" --skip-ssl > /dev/null 2>&1; do
to:
while ! ${client} --defaults-file=/tmp/defaultsfile.cnf -e "SELECT 1" > /dev/null 2>&1; do

I think the correct should be enable SSL on mariadb, someone could help on this ?

**Comment by ktx-kirtan** [comment]
I'm also stuck on the same thing

**Comment by Tecnobutrul** [comment]
Fixed on 1.1.2
